include/libtorrent/torrent_info.hpp

```diff
@@ -450,37 +452,36 @@ namespace libtorrent
 
 		// ``merkle_tree()`` returns a reference to the merkle tree for this
 		// torrent, if any.
-		//
+		std::vector<sha1_hash> const& merkle_tree() const { return m_merkle_tree; }
```

Comment:
my idea was to have separate comments for every methods. While I type class/struct method IDE usually popup hint with description. It takes from comment above method declaration.